build/Microsoft.DotNet.Cli.Signing.targets

```diff
@@ -0,0 +1,199 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <!-- This will be overridden if we're building with MicroBuild. -->
+  <Target Name="SignFiles">
+    <Message Text="Fake sign target.  Would sign: @(FilesToSign)" />
+  </Target>
+
+  <Target Name="SetSignFilesOutDir" BeforeTargets="SignFiles">
+    <PropertyGroup>
+      <OutDir Condition="'$(OutDir)' == ''">$(OutputDirectory)</OutDir>
+      <IntermediateOutputPath Condition="'$(IntermediateOutputPath)' == ''">$(IntermediateDirectory)</IntermediateOutputPath>
+    </PropertyGroup>
+  </Target>
+
+  <Import Project="$(RepoRoot)\build_tools\MicroBuild.Core.props" />
+  <Import Project="$(RepoRoot)\build_tools\MicroBuild.Core.targets" />
+
+  <Target Name="PostCompileSign" DependsOnTargets="GetPostCompileSignFiles">
+    <CallTarget Targets="SignFiles" />
+  </Target>
+
+  <Target Name="GetPostCompileSignFiles" DependsOnTargets="Init">
+    <ItemGroup>
+      <!-- External files -->
+      <FilesToSign Include="$(OutputDirectory)sdk/**/Newtonsoft.Json.dll">
+        <Authenticode>$(ExternalCertificateId)</Authenticode>
```

Comment:
It looks like the "metadata as attributes" feature was supported starting with MSBuild 15, and currently the build agents don't have MSBuild 15.